Microsoft.Dynamics365.UIAutomation.Browser/BrowserDriverFactory.cs

```diff
@@ -52,7 +52,7 @@ private static IWebDriver CreateLocalDriver(BrowserOptions options)
                 case BrowserType.Firefox:
```

Comment:
I like this additional functionality.  Any chance we can separate these changes into it's own PR?   I would like to get this in while we work through the ClosePendingEmail dialog changes.

Reply:
Yes I did it accidentally (I'm a little bit new here :) ).

Reply:
@dtu11 Done